contrib/babelfishpg_common/src/collation.c

```diff
@@ -119,6 +119,34 @@ coll_translate_t coll_translations[] =
 };
 #define TOTAL_COLL_TRANSLATION_COUNT (sizeof(coll_translations)/sizeof(coll_translations[0]))
 
+coll_translate_t coll_translations_tsql[] =
```

Comment:
let's give a meaningful name for this reverse translation, reverse_coll_translations. Also, add a comment about purpose of this C table. 

Plus, ensure that this info is sorted based on first column...

Reply:
We have seen cases in the past that maintaining self-sorted ordered arrays could lead to issues. If we extend this array to have new translations and the developer doesnt add it in the right order, how can we ensure that the test fails in all cases? Can we add negative test case?

Reply:
The test would fail in a case where the developer adds a new translation for TSQL collation T which is not in the right order and the original DDL has that particular TSQL collation T, then the exported DDL will not find the translation and give the BBF collation name instead of the TSQL name which will result in test failure.